### DISCUSSION TITLE
Very hard to find failing node!

### DISCUSSION THREAD

**Discussion by ansorre** [comfyanonymous/ComfyUI#5641] [Ideas]
A way to fast find and go to the node that failed would be greatly apreciated in ComfyUI. Unfortunately when studying very big and complez workflows usually not moda by onself but found on internet, when an error occurs you could spend even tens of minutes to find what is the node that caused the error. What a pity. A button that hides all correct nodes and leaves visible only those failing would be sooooo good!

**Comment by LukeG89** [comment]
When an error occurs, the border of the failed node turns purple or red. It should be visible enough to find it.

**Reply by LukeG89** [reply]
You can try using "follow execution" (a feature from [ComfyUI-Custom-Scripts](https://github.com/pythongosssss/ComfyUI-Custom-Scripts))


https://github.com/user-attachments/assets/63c01f1b-13b7-41a6-a329-ab653784b4c1

**Reply by ansorre** [reply]
> When an error occurs, the border of the failed node turns purple or red. It should be visible enough to find it.

Good. Now imagine to find it in 2528 other nodes.

**Comment by ltdrdata** [comment]
Please move this request to front repo.
https://github.com/Comfy-Org/ComfyUI_frontend
